action.yml

```diff
@@ -12,6 +12,11 @@ inputs:
     description: "Your Wakatime API Key"
     required: true
 
+  API_BASE_URL:
+    description: "Aternative API base URL when using a third-party WakaTime backend"
+    default: "https://wakatime.com/api"
```

Comment:
Pardon my ignorance about other metric tools, can they all work on WakaTime API, or isn't it true that others provide different API keys?

Reply:
Of course the API keys themselves are different, i.e. you can't access Wakapi with your WakaTime key. However, their formats and the way they're passed to the API are exactly the same, as those services also need to be compatible with WakaTime's client-side tooling (i.e. IDE plugins and `wakatime-cli`). Does this answer your question?

Reply:
Ah I see, thanks for clarifying, and the PR too!